index.js

```diff
@@ -1,18 +1,62 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+const header = document.querySelector('h1')
+// console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = 'blue'
 
 /***** Deliverable 3 *****/
 // console.log('This is what the traveler object looks like: ', traveler)
+//1. Find the element we're interested in
+const profile = document.querySelector('div#profile')
+//2. update the property of that element
+const img = profile.querySelector('img')
+img.src = traveler.photo 
+img.alt = traveler.name
+
+const h2 = profile.querySelector('h2')
+h2.textContent = traveler.name
+
+const em = profile.querySelector('em')
+em.textContent = traveler.nickname
+
```

Comment:
Great!! Glad you included the alt attribute! 👏 

Would like a little more descriptive variable names for the elements.

```
const travelerDiv = document.querySelector('div#profile')

const travelerImg = travelerDiv.querySelector('img')
travelerImg.src = traveler.photo
travelerImg.alt = traveler.name

const travelerNameH2 = travelerDiv.querySelector('h2')
travelerNameH2.textContent = traveler.name 

const travelerNicknameEm = travelerDiv.querySelector('em')
travelerNicknameEm.textContent = traveler.nickname 
```